probe-rs/src/probe/ftdi/mod.rs

```diff
@@ -378,13 +384,15 @@ impl JtagAdapter {
         }
 
         // Write IR register
+        // Do we always have to write the IR reg? JLink impl checks cached value before attempting
```

Comment:
We could probably do that here as well, to optimize it. But best would be if we can find a common high level implementation, so that we only have to implement it once.